subsys/bootloader/cmake/sign.cmake

```diff
@@ -196,3 +196,24 @@ foreach (slot ${slots})
     )
 
 endforeach()
+
+if (CONFIG_BUILD_S1_VARIANT)
+
+  include(${NRF_DIR}/cmake/fw_zip.cmake)
+
+  # Generate zip file with both update candidates
+  set(s0_bin_path ${PROJECT_BINARY_DIR}/signed_by_b0_s0_image.bin)
+  get_filename_component(s0_name ${s0_bin_path} NAME)
+  set(s1_bin_path ${PROJECT_BINARY_DIR}/signed_by_b0_s1_image.bin)
+  get_filename_component(s1_name ${s1_bin_path} NAME)
+
+  generate_dfu_zip(
+    ${PROJECT_BINARY_DIR}/dfu_b0.zip # Zip out
```

Comment:
I think this name should indicate what the zip contains.

Reply:
Updated to `dfu_application.zip` which is the same value used in `mcuboot.cmake`